### DISCUSSION TITLE
Uncaught ReferenceError: Livewire is not defined

### DISCUSSION THREAD

**Discussion by Djagannath** [opcodesio/log-viewer#188] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/22425146/217285634-49bc960f-d46f-4578-b899-e92f6024e732.png)

livewire library not loading in server
 
Got error. Help me pls.

**Comment by arukompas** [comment]
Hey @Djagannath , can you please tell me more about the environment? What OS, PHP version, route of the log viewer you've setup, etc. The more the better.

**Reply by Djagannath** [reply]
I use centos, php 8 1. Change rout in config to virw-logs

**Reply by Djagannath** [reply]
I see livewires in vendor. But it dose not load

**Reply by Djagannath** [reply]
php artisan livewire:publish 
solved a problem

**Reply by arukompas** [reply]
Great to know you've solved it @Djagannath ! 🙌 I'm planning of moving away from Livewire in the future to avoid problems like this. The package should just work from the start without any additional commands.

**Comment by arukompas** [comment, marked as answer]
Log Viewer v2 no longer uses Livewire, thus this should no longer be an issue in Log Viewer v2 and higher.
